libcodechecker/libhandlers/store.py

```diff
@@ -254,10 +256,15 @@ def collect_file_hashes_from_plist(plist_file):
                           .format(f))
                 shutil.copyfile(f, target_file)
 
+        hashes_fname = "content_hashes.json"
+        hashes_fpath = os.path.join(temp_dir, hashes_fname)
+        with open(hashes_fpath, "w") as f:
+            json.dump(file_to_hash, f)
+
         with zipfile.ZipFile(zip_file, 'w') as zipf:
             for root, _, files in os.walk(temp_dir):
-                for file in files:
-                    f = os.path.join(root, file)
+                for sf in files:
```

Comment:
Why was a rename `file` to `sf` needed?

Reply:
file is a builtin and its not nice to use builtins for var names.